### ISSUE TITLE
Error at "Executing load_dotnet40"

### ISSUE THREAD

**Issue by apalazzi** [rockerbacon/modorganizer2-linux-installer#146]
Hi,

I'm trying to install mod organizer 2+skyrim from Lutris; after solving the problem in issue #61 I have another error: a window pops up with the message "Error while installing winetricks, please run Lutris from a terminal and check the logs".
The logs in the terminal however don't say much:

```Executing w_do_call vcrun2019
Executing mkdir -p /home/andrea/.local/share/lutris/runners/winesteam
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
vcrun2019 already installed, skipping
Executing w_do_call dotnet40
Executing mkdir -p /home/andrea/.local/share/lutris/runners/winesteam
------------------------------------------------------
You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
------------------------------------------------------
Executing load_dotnet40```
And that's all, the load_dotnet40 is the last line on the terminal.

**Comment by rtentser** [comment]
Can confirm it. Same here.

**Comment by tonkonogov** [comment]
Some more logs from my terminal
```
Executing load_dotnet40 
------------------------------------------------------
warning: This package (dotnet40) is broken in wine-5.17. Broken since 5.12. See https://bugs.winehq.org/show_bug.cgi?id=49532 for more info.
------------------------------------------------------
```

The bug is claimed to be fixed in wine-5.18 (released yesterday on arch), but we need to wait until it gets into repos and then winetricks will unblock it. Alternatively, you can downgrade your wine to 5.11.

**Comment by rockerbacon** [comment]
Closing since the latest version does not apply dotnet40 to the prefix
